.gitignore

```diff
@@ -490,3 +490,6 @@ swa-cli.config.json
 
 # python devcontainer
 /python/.devcontainer/*
+
+# dotnet Verify
+**/*.received.txt
```

Comment:
TLDR:
This is to prevent `*.received.txt` from being checked into the repo.

A bit more explanation:
the intention is to avoid committing *.received.txt files to the repository. These files are generated during testing by the Verify function and are compared with corresponding *.verified.txt files. If no differences are found, the test passes and the *.received.txt file is deleted. If there are differences, the test fails, and the *.received.txt file remains. In both cases, received.txt files are temporary and should not be included in the repository since they are only used for comparison purposes

Reply:
@LittleLittleCloud can you configure the Verify function to use another directory for that purpose? 
Something like `%Temp%` for Windows OR `/tmp` or `/var/tmp` for Unix.

Reply:
It needs to be applied to all other tests that uses Verify? I still think adding `*.received.txt` to `.gitignore` is the easiest and right pattern here.

Reply:
Agree, let's go for the simple and transparent approach. If someone needs to find the `*.received.txt` it will be easy and they won't have to search through the code to see where they've been written.

Reply:
I would recommend going even more simpler and avoid using VerifyTests, and use just standard `Assert` calls from xUnit, as in our other tests.

Reply:
Resolved by removing all newly added verify tests in this PR